docs/proposals/image-promotion.md

```diff
@@ -0,0 +1,152 @@
+# Image Promotion
+
+## Abstract
+
+This proposal describes the best practices and defines patterns for promoting Docker
+images between different stages (eg. from "development" to "production").
+This proposal also describes priciples a way how to configure OpenShift to serve as
+a Continuous Delivery tool.
+
+## Constraints and Assumptions
+
+This document describes the following ways to promote images
+
+* Based on human intervention
+* Driven purely by OpenShift
+* Based on external Continuous Delivery tool (Jenkins, go.cd, Snap, ...)
```

Comment:
i still don't think the bulk of this comment has been addressed..ping me on irc if it's not clear what i'm suggesting.